tagging/apply_tags.py

```diff
@@ -69,9 +72,20 @@ def apply_tags(
         required=True,
         help="Owner of the image",
     )
+    arg_parser.add_argument(
+        "--variant",
+        required=True,
+        type=str,
```

Comment:
```suggestion
```

Not needed here, it will be string automatically, matches `--owner`.

Reply:
Fixed.